### PR TITLE
File added/removed events

### PR DESCRIPTION
**Pull request by robchia**
- Added `file-added-event`, `file-removed-event`
- Expose `cancel` and `cancelAll` methods

### CODE REVIEW

**Review by tobyond — APPROVED** [review]
👍 
did the stimulus 2 update ever get integrated? I'm still seeing the error in console.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.2] - 2021-02-03
```
Comment (trobrock) [inline CHANGELOG.md:9]: Should be 1.1.0, minor updates are reserved for bugs. https://semver.org/

File: src/index.js [hunk 2]
```diff
@@ -49,45 +49,39 @@ class DropzoneController extends Controller {
...
-      });
-      window.dispatchEvent(newEvent);
+  handleFileAdded(file) {
+    if (file.accepted) {
+      this.dispatchEvent(this.fileAddedEvent, { detail: { file: file }});
+
+      setTimeout(() => {
+        const controller = new DirectUploadController(this, file);
+        controller.start();
+      }, 3500);
```
Comment (trobrock) [inline src/index.js:59]: What is the delay for?

**Review by trobrock — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (robchia) [reply src/index.js:59]: I don't quite remember-- removing it

**Comment by robchia** [comment]
@tobyond Yeah that should've been updated, it's correct in dev, but not sure why it's still rendering the old "data-target" in prod

**Review by trobrock — APPROVED** [review]

**Pull request merged**